### DISCUSSION TITLE
Should we release 0.70.3?

### DISCUSSION THREAD

**Discussion by kelset** [reactwg/react-native-releases#36] [Patches]
### Released, please refer to #38
---
Current Release: [0.70.2](https://github.com/facebook/react-native/releases/tag/v0.70.2)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

---

#### Issues we are aware of

1. https://github.com/facebook/react-native/issues/34656
    * this will be addressed in Metro 73 in RN71, by this commit https://github.com/facebook/metro/commit/c2365bb1d72a3773b31c05feab13a96afac484df
2. https://github.com/facebook/react-native/issues/34673
    * a workaround exists, a PR is in the works: https://github.com/facebook/react-native/pull/34826
4. https://github.com/facebook/react-native/issues/34665
    * investigation is still ongoing

#### List of qualified picks

1. https://github.com/facebook/react-native/commit/191eb62edb58bc8e934226f4cbd4829c88958455
2. https://github.com/facebook/react-native/commit/85c0c0f21fdb52543e603687a3c42dc40dff572b
3. https://github.com/facebook/react-native/commit/9d08d55bbef4e79a8843deef90bef828f7b9a6ef
4. https://github.com/facebook/react-native/commit/967de03f304404ac8817936da37ca39514a09e33
5. https://github.com/facebook/react-native/commit/2f58e520061a31ab90f7bbeef59e2bf723605106

#### Local commits to backport to main

1.

**Comment by joe-sam** [comment]
A couple of relevant fixes for new architecture builds on Windows environment.
Use NMake generator for Hermes build on Windows ([#34790](https://github.com/facebook/react-native/pull/34790))
Fixing failure building RN codegen CLI on Windows ([#34791](https://github.com/facebook/react-native/pull/34791))

**Comment by Noitham** [comment]
This commit here reverts a commit that introduced an issue with TextInputs on 0.70
https://github.com/facebook/react-native/commit/191eb62edb58bc8e934226f4cbd4829c88958455



I've described the issue here: https://github.com/facebook/react-native/issues/34862

**Reply by wfern** [reply]
This commit(revert) fixes this issue introduced on 0.70 too: https://github.com/facebook/react-native/issues/34668

**Comment by kelset** [comment]
we'll have to cherry pick this to address Xcode 14: https://github.com/facebook/react-native/pull/34826

**Reply by kelset** [reply]
commit: https://github.com/facebook/react-native/commit/967de03f304404ac8817936da37ca39514a09e33

**Comment by erfinbadrian** [comment]
`info.plist`

```
<key>CFBundleShortVersionString</key>
<string>1.0</string>
<key>CFBundleVersion</key>
<string>1</string>
```

i think we need to change that to 

```
<key>CFBundleShortVersionString</key>
<string>>$(MARKETING_VERSION)</string>
<key>CFBundleVersion</key>
<string>$(CURRENT_PROJECT_VERSION)</string>
```

to following xcode settings.

**Reply by kelset** [reply]
can you submit a PR to do this change in the template?

**Reply by erfinbadrian** [reply]
Done with this PR: [34919](https://github.com/facebook/react-native/pull/34919)

**Comment by renchap** [comment]
The `Animated.Value` bug has been fixed in [34927](https://github.com/facebook/react-native/pull/34927) and should be included

**Reply by kelset** [reply]
commit: https://github.com/facebook/react-native/commit/2f58e520061a31ab90f7bbeef59e2bf723605106
